Editor/UI/Drawers/TabsGroupDrawer.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Linq;
+using Innoactive.Creator.Core.Tabs;
+using UnityEditor;
+using UnityEngine;
+
+namespace Innoactive.CreatorEditor.UI.Drawers
+{
+    [DefaultTrainingDrawer(typeof(ITabsGroup))]
+    internal class TabsGroupDrawer : AbstractDrawer
+    {
+        public override Rect Draw(Rect rect, object currentValue, Action<object> changeValueCallback, GUIContent label)
```

Comment:
overall this method does a alot without giving me any hints what actually happens, would love to see at lease 2-3 comments or smaller methods, what is currently done.